sugoi-api-model/src/main/java/fr/insee/sugoi/model/UserStorage.java

```diff
@@ -94,4 +103,26 @@ public void setProperties(Map<RealmConfigKeys, String> properties) {
   public void addProperty(RealmConfigKeys key, String value) {
     this.properties.put(key, value);
   }
+
+  public Consumer<User> getAddUsDefinedAttributesTransformer(RealmConfigKeys configKey) {
+    return (User user) -> {
+      try {
+        if (getUserMappings() != null && getProperties().containsKey(configKey))
+          ((Map<?, ?>) jsonPropertyMapper.readValue(getProperties().get(configKey), Map.class))
```

Comment:
Avoid usage of a json format for this property
will cause too long property values 
inseepropriete: user_us_defined_attributes${'fieldexample':'Example $(username) $(mail)','fieldexample2':'tototot${usename)'}

use instead common list of properties like other lists, for example
inseepropriete: user_us_defined_attributes$fieldexample##Example $(username) $(mail)
inseepropriete: user_us_defined_attributes$fieldexample2##tototot${usename)